apiserver/plane/app/views/workspace/recent_visit.py

```
@@ -0,0 +1,31 @@
+        return WorkspaceRecentVisitSerializer
+
+    @allow_permission([ROLE.ADMIN, ROLE.MEMBER, ROLE.GUEST], level="WORKSPACE")
+    def list(self, request, slug):
+            user_recent_visits = UserRecentVisit.objects.filter(workspace__slug=slug)
+
+            entity_name = request.query_params.get("entity_name")
+
+            if entity_name: 
+                user_recent_visits = user_recent_visits.filter(entity_name=entity_name)
+                
+            user_recent_visits = user_recent_visits.filter(entity_name__in=["issue","page","project"])
```

Ruff (0.8.2)

Line too long (102 > 88)

(E501)